Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ureq"
-version = "2.0.2"
+version = "2.0.3"
```

Comment:
Oh, one thing: this should actually take us to `2.1.0`. https://semver.org/

> Given a version number MAJOR.MINOR.PATCH, increment the:
>    MAJOR version when you make incompatible API changes,
>    MINOR version when you add functionality in a backwards compatible manner, and
>    PATCH version when you make backwards compatible bug fixes.

Reply:
good catch!